depends/builders/freebsd.mk

```
@@ -1,5 +1,5 @@
 build_freebsd_CC=clang
 build_freebsd_CXX=clang++
 
-build_freebsd_SHA256SUM = shasum -a 256
+build_freebsd_SHA256SUM = sha256sum
 build_freebsd_DOWNLOAD = curl --location --fail --connect-timeout $(DOWNLOAD_CONNECT_TIMEOUT) --retry $(DOWNLOAD_RETRIES) -o
```

```suggestion
build_freebsd_DOWNLOAD = fetch --timeout 180 --retry -o
```

related but probably out-of-scope: freebsd's `fetch` utility could be used to also drop the curl dependency, there is a regression since it is unable to limit number of retries([16-year old patch awaits feedback](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=131427)), and to set a handshake timeout, but these can be approximated with a single connection timeout